src/app/application.h

```diff
@@ -89,6 +89,11 @@ class Application final : public BaseApplication
 
     const QBtCommandLineParameters &commandLineArgs() const;
 
+#ifdef Q_OS_WIN
+    int memoryWorkingSetLimit() const;
+    void setMemoryWorkingSetLimit(int size);
```

Comment:
Sorry I gave the wrong suggestion before. This should store bytes and preferably use `qint64`. The UI can still use `MiB`. And webapi will use bytes.

Reply:
> This should store bytes and preferably use `qint64`.

Why?

Reply:
So it is consistent with other settings (from the viewpoint of webapi and user-editable settings file). `int` is ok if users thinks 4GiB limit is acceptable.

Reply:
No big deal but... 

1. We clearly do not need such precision and it is unlikely that we will ever need it.
2. Having an inconsistency of units between the config file and the UI, we will definitely receive complaints from some "smart guys".

How should we handle the cases if there is more precise value is stored in config than we can represent in UI? If we just discard some less significant part (or round it up) the value will be rewritten n config when applying preference changes in UI.

Reply:
>We clearly do not need such precision and it is unlikely that we will ever need it.

It is all about being consistent... There was similar problems in the past and IIRC this was the least problematic way.

>Having an inconsistency of units between the config file and the UI, we will definitely receive complaints from some "smart guys".

I don't remember there was such complaint.

>How should we handle the cases if there is more precise value is stored in config than we can represent in UI? If we just discard some less significant part (or round it up) the value will be rewritten n config when applying preference changes in UI.

I would just round them. Users will just have to live with it. 😛

Reply:
> > Having an inconsistency of units between the config file and the UI, we will definitely receive complaints from some "smart guys".
> 
> I don't remember there was such complaint.

I would complain if i edited the config file manually.

Reply:
@Chocobo1 
After some consideration, I would still leave using megabytes (@an0n666 yes, yes, I belong to the "old school", so I mean mebibytes, if you are more used to it). Using bytes will not bring any benefit, but only problems (I don't see any benefit in consistency if it is an end in itself). Maybe you know something that is not obvious to me?